docs/tutorial.md

```diff
@@ -3,7 +3,7 @@
 !!! note
     Did you find any of these instructions confusing? [Edit this file] and submit a pull request with your improvements!
 
-[Edit this file]: https://github.com/arup-group/cookiecutter-pypackage/blob/main/docs/tutorial.md
+[Edit this file]: https://github.com/arup-group/pypackage-template/blob/main/docs/tutorial.md
```

Comment:
Again - this URL doesn't currently exist. Just flagging that; if there's a plan for that, cool.

Reply:
Chicken and egg problem, I suppose. When to rename the repo, after or before this PR is merged?